packages/fxa-content-server/app/scripts/lib/survey-filter.js

```diff
@@ -350,11 +350,18 @@ export const createSurveyFilter = (
   return async (surveyConfig) =>
     !!(
       surveyConfig &&
-      surveyConfig.rate &&
-      withinRate(surveyConfig.rate) &&
+      parseInt(surveyConfig.rate) >= 0 &&
```

Comment:
I'm parsing this value because 0 is falsy and was giving false negatives when the rate was set to 0. I believe this is still safe because `withinRate` handles the actual rate check directly below.